neo/NeoSystem.cs

```diff
@@ -56,9 +56,10 @@ public void StartNode(int port = 0, int ws_port = 0)
             });
         }
 
-        public void StartRpc(IPAddress bindAddress, int port, Wallet wallet = null, string sslCert = null, string password = null, string[] trustedAuthorities = null)
+        public void StartRpc(IPAddress bindAddress, int port, Wallet wallet = null, string sslCert = null, string password = null, 
+            string[] trustedAuthorities = null, long maxGasInvoke = 0L)
```

Comment:
@erikzhang you probably want this changed to Fixed8 too then I suppose. Feel free to change it and be merge. I’m not at my computer for a while.

Reply:
I went back to computer and changed it.